nemo/tron/setup.py

```diff
@@ -164,11 +173,14 @@ def setup(
 
     # Data stuff.
     timers("train/valid/test-data-iterators-setup", log_level=0).start(barrier=True)
+    if "tokenizer" in inspect.signature(train_valid_test_datasets_provider).parameters:
```

Comment:
could you describe this check more?

Reply:
The pretraining dataset provider doesnt require tokenizer since its already preprocessed.